core/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ## [Unreleased] - ReleaseDate
 
+### Added
+
+- **(breaking)** [#552](https://github.com/embedded-graphics/embedded-graphics/pull/552) Added the `Output` associated type to `Drawable` to return intermediate values from drawing operations.
```

Comment:
Should we specify that the returned value is an "intermediate value"? IMO a `Drawable` should be allowed to return any value that might be useful to the caller.